tests/scripts/test_pkgdev_bugs.py

```diff
@@ -0,0 +1,91 @@
+import itertools
+import os
+import sys
+import textwrap
+from types import SimpleNamespace
+from unittest.mock import MagicMock
+
+import pytest
+from pkgcore.ebuild.atom import atom
+from pkgcore.test.misc import FakePkg
+from pkgdev.scripts import pkgdev_bugs as bugs
+from snakeoil.formatters import PlainTextFormatter
+from snakeoil.osutils import pjoin
+
+def mk_pkg(repo, cpvstr, maintainers, **kwargs):
+    kwargs.setdefault("KEYWORDS", ["~amd64"])
+    pkgdir = os.path.dirname(repo.create_ebuild(cpvstr, **kwargs))
+    # stub metadata
+    with open(pjoin(pkgdir, 'metadata.xml'), 'w') as f:
+        f.write(textwrap.dedent(f"""\
+            <?xml version="1.0" encoding="UTF-8"?>
+            <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
```

Comment:
Know it's only minor, but:
```suggestion
            <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
```